examples/search_embeddings/pyproject.toml

```diff
@@ -9,7 +9,6 @@ license = "Apache-2.0"
 python = ">=3.8,<3.12"
 kolena = ">=0.94.0,<1"
 s3fs = "^2023.5.0"
-kolena-embeddings = {path = "local_packages/kolena_embeddings.tar.gz"}
```

Comment:
Do we need to add exactly those configs as described in readme here?

Reply:
Added the package back here for completeness